spanner/transaction.go

```diff
@@ -185,6 +185,9 @@ type ReadOptions struct {
 
 	// An option to control the order in which rows are returned from a read.
 	OrderBy sppb.ReadRequest_OrderBy
+
+	// A lock hint mechanism to use for this request.
```

Comment:
Lets mention that this is applicable only for RW transactions?

Reply:
Done